generated/nifgen/nifgen.proto

```diff
@@ -1628,7 +1682,7 @@ message GetStreamEndpointHandleResponse {
 
 message ImportAttributeConfigurationBufferRequest {
```

Comment:
For this and ExportAttributeConfigurationBuffer param types changing, we had to update one of the tests: https://github.com/ni/grpc-device/pull/747/files#diff-d73e0dbb4de64077825e682dcf4d1c1d094acfeaa22889d68ecb0c7362d0a5a8